config.cmake

```diff
@@ -58,7 +58,9 @@ HHVM_EXTENSION(mongodb
  libmongoc/src/mongoc/mongoc-gridfs-file.c libmongoc/src/mongoc/mongoc-gridfs.c
  libmongoc/src/mongoc/mongoc-host-list.c
  libmongoc/src/mongoc/mongoc-index.c libmongoc/src/mongoc/mongoc-init.c
- libmongoc/src/mongoc/mongoc-list.c libmongoc/src/mongoc/mongoc-log.c
+ libmongoc/src/mongoc/mongoc-list.c
+ libmongoc/src/mongoc/mongoc-linux-distro-scanner.c
+ libmongoc/src/mongoc/mongoc-log.c
```

Comment:
Unrelated, but why were mongoc-index.c and mongoc-init.c on the same line? It looks like you only intended to put related files on the same line (e.g. gridfs, matcher).